src/istio/chart/templates/tls-cert.yaml

```
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
 
 {{- $tls := .Values.tls }}
-{{ if $tls.cert }}
+{{ if and $tls.cert (not .Values.tls.credentialName) }}
```

```suggestion
{{ if and $tls.cert (not $tls.credentialName) }}
```

Definitely a nit, just a suggested change for consistency with values references.